hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -97,8 +97,10 @@ def _default_employee(self):
     )
     line_ids = fields.One2many(
         comodel_name='hr_timesheet.sheet.line',
+        inverse_name='sheet_id',
         compute='_compute_line_ids',
         string='Timesheets',
+        store=True,
```

Comment:
Is `store` relevant for recordsets?

Reply:
Yes, it's an ingredient to my black magic 😎

Reply:
I wonder if it's possible to do one PR for black magic and other one for negative number thing